cl/opinion_page/templates/includes/de_list.html

```diff
@@ -171,6 +171,26 @@
                             target="_blank"
                             rel="nofollow">Buy on PACER {% if rd.page_count %}(${{ rd|price }}){% endif %}
                         </a>
+                          {% if request.user.is_authenticated %}
+                          <a href="{% url 'create_prayer' rd.id %}"
+                            class="btn btn-default btn-xs prayer-button {% if rd.prayer_exists or not prayer_eligible %}disabled{% endif %}"
+                            data-document-id="{{ rd.id }}"
+                            {% if rd.prayer_exists %}
+                              title="You have already requested this document."
+                            {% elif not prayer_eligible %}
+                              title="You have reached your daily request limit."
+                            {% endif %}
+                            style="margin-left: {% if rd.page_count %}8px{% else %}44px{% endif %};">
```

Comment:
I am not a fan of hardcoding the amount of pixels to leave between the two buttons. Open to suggestions for improving it.

Reply:
Yeah, this seems hard. I think this would require some custom CSS where you wrap both buttons within a div using a class with `display: table;`. Then, on each button, use `display: table-cell;` and adjust the buttons' widths to sizes that make them look aligned.

Do you want to try it? 

btw custom css can be added in: `override.css`

Reply:
We're redoing all of our HTML very soon, I'm frightened to say, so the normal reasons for separating out things like this might not apply. I've never allowed inline CSS before, but maybe this is fine?